api/v1/cluster_webhook_test.go

```diff
@@ -1440,6 +1441,8 @@ var _ = Describe("validate image name change", func() {
 			Expect(clusterNew.validateImageChange(&clusterOld)).To(HaveLen(1))
 		})
 		It("doesn't complain going from default imageName to same major imageCatalogRef", func() {
+			majorDefaultVersion, err := strconv.ParseFloat(utils.GetImageTag(versions.DefaultImageName), 32)
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
I think that to unit test this function it's better if we isolate ourselves from the DefultImage.
The parsing here could fail, and that would tell us nothing about the capacity to upgrade
from imageName to imageCatalogRef.